.vsts-ci.yml

```diff
@@ -119,6 +119,9 @@ jobs:
   strategy:
     maxParallel: 3
     matrix:
+      rpkg:
+        TASK: r-package
+        R_WINDOWS_VERSION: 3.6.3
```

Comment:
I moved this over from #2936 . It'll work once that PR is merged, and that PR should definitely be merged before we do this one.